src/Symfony/Component/VarDumper/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Add support for dumping virtual properties
```

Comment:
```suggestion
 * Add support for virtual properties
```
?

Reply:
Is it named virtual or hooked property?

Reply:
Even if virtual properties are related to hooks, there's no mention of the `hooked` word when using the Reflection API. I updated the code to only refer to "virtual properties"